### DISCUSSION TITLE
What happened to `notFound` etc?

### DISCUSSION THREAD

**Discussion by piotrkulpinski** [sergiodxa/remix-utils#273] [Q&A] [answered]
Hi, 
I wasn't using Remix for a while and just getting back to it. 
I tried running my last project with the latest package versions but it's failing due to removal of helper functions like `notFound ` or `serverError`. 

Can I ask why they were removed from the package and what should I use instead? 
Returning Response with proper status is the recommended way?

Thanks

**Comment by sergiodxa** [comment, marked as answer]
The `notFound`, `badRequest`, etc. function were removed in v7. They were just wrappers of `json` and setting one specific status code.

You can find them on [code for v6.6.0](https://github.com/sergiodxa/remix-utils/blob/v6.6.0/src/server/responses.ts) if you want to copy them into your own app.
